app/platform/log.go

```diff
@@ -78,6 +80,13 @@ func (ps *PlatformService) initLogging() error {
 		}
 	}
 
+	if ps.DebugBar.IsEnabled() {
+		err := ps.logger.AddTarget(debugbar.NewDebugBarLogTarget(ps.DebugBar), "debugbar", &debugbar.DebugBarLogFilter{}, &debugbar.DebugBarLogFormatter{}, logr.DefaultMaxQueueSize)
```

Comment:
I want to point out that when DebugBar is enabled, every single log line generated by the app will get published over websocket, including debug and trace.  I've not looked deeply into the websocket layer but can it handle those volumes without blocking application events?

Reply:
It is working fine in my development environment, and because it is mainly for development I think is just fine for now.

Reply:
I might be missing something, but it appears that when DebugBar is enabled every client will receive every event, including every log line.  I guess this is never intended to be turned on outside a small developer instance?